src/akkudoktoreos/utils/visualize.py

```diff
@@ -1,6 +1,8 @@
+import datetime
```

Comment:
To consistently use pendulum in Akkudoktor-EOS (and get rid of datetime) you could use:

```Python
from  akkudoktoreos.utils.datetimeutil import to_datetime
...
current_time = to_datetime(as_string=True)
```

If you want a special format of the string use the [Pendulum Formatter](https://pendulum.eustace.io/docs/#formatter), e.g.:

```Python
current_time = to_datetime(as_string="YYYY-MM-DD HH:mm:ss")
```